platform/viewer/src/appInit.js

```diff
@@ -55,7 +56,8 @@ function appInit(appConfigOrFunc, defaultExtensions) {
     DisplaySetService,
     ToolBarSerivce,
     ViewportGridService,
-    HangingProtocolService
+    HangingProtocolService,
+    CineService
```

Comment:
So many services. I think this makes sense for our current pattern, but worth noting @JamesAPetts, how do we stop services from ballooning? What other patterns/strategies can we lean on?